### ISSUE TITLE
Carousel Previous and Next controls are only in English

### ISSUE THREAD

**Issue by michael-milette** [trema-tech/moodle-theme_trema#93]
The Previous and Next links, which are only available when using a screen reader, are only in English. They should be in the language of the Moodle UI.

**Comment by rmady** [comment]
Hi @michael-milette,
Thank you for this contribution lol

Kind regards,
Rodrigo Mady

**Comment by michael-milette** [comment]
You're welcome! Thank you!
-------- Original message --------From: Rodrigo Mady ***@***.***> Date: 2022-01-21  6:51 p.m.  (GMT-05:00) To: trema-tech/moodle-theme_trema ***@***.***> Cc: Michael Milette ***@***.***>, Mention ***@***.***> Subject: Re: [trema-tech/moodle-theme_trema] Carousel Previous and Next
  controls are only in English (Issue #93) 
Hi @michael-milette,
Thank you for this contribution lol
Kind regards,
Rodrigo Mady
